src/pages/home/report/ReportActionsView.js

```diff
@@ -420,15 +432,17 @@ class ReportActionsView extends React.Component {
                             onClick={this.scrollToBottomAndUpdateLastRead}
                             onClose={this.hideFloatingMessageCounter}
                         />
-                        <ReportActionsList
-                            report={this.props.report}
-                            onScroll={this.trackScroll}
-                            onLayout={this.recordTimeToMeasureItemLayout}
-                            sortedReportActions={this.sortedReportActions}
-                            mostRecentIOUReportSequenceNumber={this.mostRecentIOUReportSequenceNumber}
-                            isLoadingReportActions={this.props.isLoadingReportActions}
-                            loadMoreChats={this.loadMoreChats}
-                        />
+                        <Animated.View style={[styles.flex1, {opacity: this.state.fadeInAnimation}]}>
```

Comment:
It seems that this animation does not depend on any parameter so why not move this inside the `ReportActionsList`. Any specific reason to put this here?

Reply:
No specific reason. Can move.

Reply:
This is fine here.

Reply:
Ohh I already moved but.

Reply:
That's fine too.